drivers/adxl345/adxl345.c

```diff
@@ -31,8 +31,8 @@
 
 #define I2C_SPEED           I2C_SPEED_NORMAL
 
-#define BUS                 (dev->i2c)
-#define ADDR                (dev->addr)
+#define BUS                 (dev->params->i2c)
```

Comment:
I think it should be `dev->params->i2c` instead of `dev->params->i2c`, same issue on the line below.

Reply:
`params` is a pointer, hence `params->i2c`